### PR TITLE
refactor: divide load() for each currency data source

### PR DESCRIPTION
**Pull request by gsegatti**
Hello, folks! This is my first PR around here, so I tried to do something simple. Ended up being bigger than I thought.
Please have a go at it and let me know if you have any suggestions.

# Description
- Just a small refactor of `load()` in `/cryptocyrrency/cryptocurrency_helpers.py`
    - Created a function for each of the sources we can retrieve data from, downsizing `load()` itself.

### CODE REVIEW

**Comment by gsegatti** [comment]
I'm afraid the `Check validity of reserved arguments` lint isn't passing but it is quite late and I'll check it tomorrow.

**Comment by gsegatti** [comment]
Seems like all the CI tests passed, only missing a tag apparently.

**Comment by jmaslek** [comment]
@jose-donato can you take a look :)

**Comment by DidierRLopes** [comment]
Amazing @gsegatti !!

File: openbb_terminal/cryptocurrency/cryptocurrency_helpers.py [hunk 1]
```diff
@@ -304,6 +304,172 @@ def get_coinpaprika_id(symbol: str):
...
 
 
+def load_from_ccxt(
+    symbol: str,
+    start_date: datetime = (datetime.now() - timedelta(days=1100)),
+    interval: str = "1440",
+    exchange: str = "binance",
+    vs_currency: str = "usdt",
+) -> pd.DataFrame:
+    """Load crypto currency data from CCXT.
```
Comment (jose-donato) [inline openbb_terminal/cryptocurrency/cryptocurrency_helpers.py:314]: we normally do here like this:

`Load crypto currency data [Source: https://github.com/ccxt/ccxt]`

File: openbb_terminal/cryptocurrency/cryptocurrency_helpers.py [hunk 2]
```diff
@@ -304,6 +304,172 @@ def get_coinpaprika_id(symbol: str):
...
+        return df
+    return df
+
+
+def load_from_coingecko(
+    symbol: str,
+    start_date: datetime = (datetime.now() - timedelta(days=1100)),
+    vs_currency: str = "usdt",
+) -> pd.DataFrame:
+    """Load crypto currency data from Coin Gecko.
```
Comment (jose-donato) [inline openbb_terminal/cryptocurrency/cryptocurrency_helpers.py:361]: Same here 
`Load crypto currency data [Source: https://www.coingecko.com/]`

File: openbb_terminal/cryptocurrency/cryptocurrency_helpers.py [hunk 3]
```diff
@@ -304,6 +304,172 @@ def get_coinpaprika_id(symbol: str):
...
+
+
+def load_from_yahoofinance(
+    symbol: str,
+    start_date: datetime = (datetime.now() - timedelta(days=1100)),
+    interval: str = "1440",
+    vs_currency: str = "usdt",
+    end_date: datetime = datetime.now(),
+) -> pd.DataFrame:
+    """Load crypto currency data from Yahoo Finance.
```
Comment (jose-donato) [inline openbb_terminal/cryptocurrency/cryptocurrency_helpers.py:412]: `Load crypto currency data [Source: https://finance.yahoo.com/]`

**Review by jose-donato — CHANGES_REQUESTED** [review]
Welcome aboard 🚀 

Everything looks good code-wise apart from the minor changes I commented. Can test it locally later today

Thanks for your contribution, if you want to contribute on more stuff related to crypto let me know

**Comment by gsegatti** [comment]
Thanks, @jose-donato, I will! 
I'll also keep in mind the way of formatting Source related comments. I've Fixed what you mentioned so far.

**Review by jose-donato — APPROVED** [review]
all good! way more clean, nice job 🚀

**Pull request merged**